src/reynir_correct/errtokenizer.py

```diff
@@ -1757,12 +1757,13 @@ def only_suggest(token: CorrectToken, m: Sequence[BIN_Meaning]) -> bool:
                 w, m = db.lookup_word(
                     corrected_txt, at_sentence_start=at_sentence_start
                 )
-                if (token.txt[0].lower() == "ó" and corrected_txt == token.txt[1:]) or (
-                    corrected_txt[0].lower() == "ó" and token.txt == corrected_txt[1:]
-                ):
-                    # The correction simply removed or added "ó" at the start of the
-                    # word: probably not a good idea
-                    pass
+                if len(corrected_txt) > 1:
```

Comment:
This does not work as expected - the conditions from the two if statements must be merged into a single condition in a single if statement. As is, all words longer than 1 character will be handled by the enclosing if statement, and will never be passed into the following `elif`.

Reply:
OK, I'll reverse this